test/extended/quota/resourcequota.go

```diff
@@ -216,6 +217,89 @@ var _ = g.Describe("[sig-api-machinery][Feature:ResourceQuota]", func() {
 			})
 			o.Expect(err).NotTo(o.HaveOccurred())
 		})
+
+		g.It("when exceed openshift.io/image-tags will ban to create new image references in the project [Skipped:Disconnected]", func() {
+			testProject := oc.SetupProject()
+			testResourceQuotaName := "my-image-tag-quota-" + testProject
```

Comment:
You don't need to randomize resource name, limit ranges are namespaced, so they are unique in a particular namespace.

Reply:
Updated